addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -508,18 +508,7 @@ var FormRenderer = BasicRenderer.extend({
      * @returns {jQueryElement}
      */
     _renderStatButton: function (node) {
-        var $button = $('<button>').addClass('btn btn-sm oe_stat_button');
-        if (node.attrs.icon) {
-            $('<div>')
-                .addClass('fa fa-fw o_button_icon')
-                .addClass(node.attrs.icon)
-                .appendTo($button);
-        }
-        if (node.attrs.string) {
-            $('<span>')
-                .text(node.attrs.string)
-                .appendTo($button);
-        }
+        var $button = this._renderButtonFromNode(node); // FIXME force oe_stat_button ?
```

Comment:
The previous code forced the 'oe_stat_button' class, the new code supposes it is explicitly set in the view... is this a problem ?

Reply:
maybe... this should probably not even appear in views' arch, and should be automatically added by the webclient